scripts/generate_dump

```diff
@@ -1005,6 +1005,31 @@ save_counter_snapshot() {
     save_cmd_all_ns "ifconfig -a" "ifconfig.counters_$idx"
 }
 
+###############################################################################
+# save the debug dump output
+###############################################################################
+save_dump_state_all_ns() {
+    MODULES="$(dump state -s | sed '1d;2d' | awk '{print $1}')"
+    local UVDUMP="unified_view_dump"
+    echo "DEBUG DUMP: Modules Available to Generate Debug Dump Output"
+    echo $MODULES
+    $MKDIR $V -p $LOGDIR/$UVDUMP
+
+    for addr in $MODULES;
+    do
+            save_cmd_all_ns "dump state $addr all --key-map" "$UVDUMP/$addr"
```

Comment:
save_cmd_all_ns is to run this command for all asics why then we need to have the below loop?
Would save_cmd_all_ns call "dump state $addr all --key-map" NUM_ASICS times and produce the same output because noone passes --namespace parameter?

Reply:
Done